mail_activity_team/models/mail_activity_mixin.py

```diff
@@ -23,13 +23,27 @@ def _search_my_activity_date_deadline(self, operator, operand):
             return super(MailActivityMixin, self)._search_my_activity_date_deadline(
```

Comment:
`return super()._search_my_activity_date_deadline(`

Reply:
@bealdav Can this be reproduced in an earlier version?

Reply:
When the activities pop-up is "closed", it always shows **My Activities**.
It only temporarily shows the Team Activities when you select it in the pop-up.
It was designed like this so I also left this like it was.